README.md

```diff
@@ -94,32 +97,33 @@ user@host:~$ substrait-validator input.yaml --out-file output.html --mode ignore
 ```
 
 This generates `output.html`, a self-contained HTML file describing the plan.
-
 Just like the input file, the output file format is derived from the file
-extension, so the `.html` part is significant. If you don't want to rely on
-this, you can also just specify the formats you want manually using `--in-type`
-and `--out-type`.
+extension, so the `.html` part is significant.
 
-`--mode ignore` tells the validator to emit a file and exit with code 0
-regardless of the validation result. The full list of modes is:
+The validator only emits an output file if it exits with code 0, and exits
+with code 1 if validation fails, in order to play nicely with build systems
+like `make`. `--mode ignore` just tells the validator to ignore the validation
+result, so it always emits the file. The full list of modes is:
 
  - `strict`: fail unless the plan was proven to be valid;
  - `loose` (default): fail if the plan was proven to be invalid;
- - `ignore`: ignore the validation result, though the plan still needs some
-   level of sanity to succeed; for example, the file must exist, and must
-   decode according to the specified file format.
- - `convert`: don't run validation at all; simply convert between different
-   representations of the given `substrait.Plan` message. For example, you
-   can use this to convert between the binary protobuf serialization format
-   and any of the text-based formats supported by the validator.
+ - `ignore`: ignore the validation result (the plan still needs some level of
+   sanity for the validator to succeed; for example, the file must exist, and
```

Comment:
```suggestion
   sanity for the validator to succeed; for example, the file must exist and
```

Reply:
Both are grammatically correct, but the comma signals that the two things are separate examples rather than a single one. That was my intention, so I've left this one as is.